pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ModelInterpreter.groovy

```diff
@@ -82,80 +79,84 @@ public class ModelInterpreter implements Serializable {
             // Entire build, including notifications, runs in the withEnv.
             withEnvBlock(root.getEnvVars()) {
                 inWrappers(root.wrappers) {
-                // Stage execution and post-build actions run in try/catch blocks, so we still run post-build actions
```

Comment:
Hmm, I had some issues getting the indentation correct at first, through that I had fixed it but apparently not.